test/previews/primer/alpha/action_list_preview.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    # @label ActionList
+    class ActionListPreview < ViewComponent::Preview
+      # @label Sub lists
+      def sub_lists
```

Comment:
I'm still a little bit confused with `sub-list`. It feels funny to me to repeat the `<action-list>` wrapper:

![image](https://user-images.githubusercontent.com/18661030/190496987-cb28371b-ff89-45ed-b4c3-84c772f8303c.png)

Also, if we want to move back towards the entire list always being a `ul`, the divider component should become an `li` instead of a `div

Reply:
I was able to get rid of the `<action-list>` wrapper. It only renders once per list now.